@here/harp-mapview-decoder/lib/GeoJsonTiler.ts

```diff
@@ -69,7 +69,7 @@ export class GeoJsonTiler implements ITiler {
             buffer: BUFFER, // tile buffer on each side
             lineMetrics: false, // whether to calculate line metrics
             promoteId: null, // name of a feature property to be promoted to feature.id
-            generateId: true, // whether to generate feature ids. Cannot be used with promoteId
```

Comment:
this was overwritting featureIds coming from geojson data. It was used to generate sequential numbers that could be use later to match the output features with the original geojson features to add the original geometry, see below.